sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
```

Comment:
I remember this is a beta preview (correct me if I'm wrong), so remember to update the version number in change log/version.py, classifier in setup.py

Reply:
@rakshith91 , could you update the version number to be 7.6.0?